php-checks/src/test/resources/checks/HttpOnlyCheck.php

```diff
@@ -34,3 +34,19 @@
 
   setcookie(httponly: foo(false));
   setrawcookie(httponly: $httpOnly, name: $name);
+
+use Symfony\Component\HttpFoundation\Cookie;
+
+Cookie::create('name', 'value', $minutes, $path, $domain, $secure, FALSE); // Noncompliant
```

Comment:
I would suggest to test also the namedparameter case.

```suggestion
Cookie::create('name', 'value', $minutes, $path, $domain, $secure, FALSE); // Noncompliant
Cookie::create(name:'name', value:'value', httpOnly:FALSE); // Noncompliant
```

Reply:
Indeed, I was thinking that it may not be necessary since retrieving correctly the named arguments is an already tested functionality, but in the implementation, we need to specify the correct name of the argument, so it is a good idea